tools/devops/automation/templates/sign-and-notarized/setup.yml

```diff
@@ -0,0 +1,54 @@
+# template that setups all the diff plugins needed to perform the sign and notarize step
+parameters:
+
+- name: signatureType
+  type: string
+  default: 'Real'
+
+- name: condition
+  default: and(succeeded(), eq(variables['IsPRBuild'], 'False'))
+
+steps:
+
+# DO NOT USE THE checkout.yml template. The reason is that the template changes the hash which results in a problem with the artifacts scripts
+- checkout: self          # https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema#checkout
+  clean: true             # Executes: git clean -ffdx && git reset --hard HEAD
+  submodules: recursive
+  path: s/xamarin-macios
+
+- checkout: maccore
+  clean: true
+  persistCredentials: true  # hugely important, else there are some scripts that check a single file from maccore that will fail
```

Comment:
Do we really need maccore for signing? Do we even need xamarin-macios?

Reply:
Wedefinetly need xamarin-macios because of the pwsh scripts that are in the scripts dir.